shared/telemetry.ts

```diff
@@ -0,0 +1,53 @@
+import * as sourcegraph from 'sourcegraph'
+
+// Polyfill
+const g: any = global
+g.performance = g.performance || { now: () => 0 }
```

Comment:
Out of curiosity, why use define `const g` instead of modifying global directly?